Add documentation for series copy, optional series and draft courses (#598)
dodona-edu/dodona-edu.github.io: nl/guides/teachers/creating-a-course/index.md

Reviewer: nit (also old): To align with the `title`

```suggestion
# Een nieuwe cursus aanmaken
```

:::

> Als je de nodige rechten hebt op Dodona, dan kan je eenvoudig zelf een cursus aanmaken. Dit doe je door op de knop `Maak een nieuwe cursus aan` te klikken in de rechterbovenhoek van het [cursusoverzicht](https://dodona.be/nl/courses). Vanaf de landingspagina bereik je dit overzicht door te klikken op `Blader door alle cursussen op Dodona`.

Reviewer: ```suggestion
Als je de nodige rechten hebt op Dodona, kan je eenvoudig zelf een cursus aanmaken. Dit doe je door op de knop `Maak een nieuwe cursus aan` te klikken in de rechterbovenhoek van het [cursusoverzicht](https://dodona.be/nl/courses). Vanaf de landingspagina bereik je dit overzicht door te klikken op `Blader door alle cursussen op Dodona`.
```

* `Lesgevers`: de naam of namen van de lesgevers van de cursus.

> * `Conceptcursus`: dit bepaalt of de cursus een concept is of niet. Conceptcursussen kunnen gebruikt worden om een cursus voor te bereiden voor die beschikbaar wordt voor studenten. Een conceptcursus kan ook gebruikt worden als sjabloon of als vragendatabank om inhoud te kopiëren naar andere cursussen. Een conceptcursus kan geen academiejaar hebben, en de instelling voor vragen en toegangsrechten kunnen ook niet veranderd worden.

Reviewer: ```suggestion
* `Conceptcursus`: dit bepaalt of de cursus een concept is of niet. Conceptcursussen kunnen gebruikt worden om een cursus voor te bereiden voordat die beschikbaar wordt voor studenten. Een conceptcursus kan ook gebruikt worden als sjabloon of als vragendatabank om inhoud te kopiëren naar andere cursussen. Een conceptcursus kan geen academiejaar hebben, en de instellingen voor vragen en toegangsrechten kunnen ook niet veranderd worden.
```

Reviewer: Or similar as above to the reworded `Draft course` explanation:

```suggestion
* `Conceptcursus`: maakt van de cursus een conceptcursus. Conceptcursussen zijn handig om een cursus voor te bereiden voordat die beschikbaar wordt voor studenten, of als sjabloon om inhoud uit te kopiëren. Twee zaken liggen vast voor een conceptcursus: er kan geen academiejaar ingesteld worden, en de instellingen voor `Vragen` en toegangsrechten kunnen niet aangepast worden.
```

> * `Conceptcursus`: dit bepaalt of de cursus een concept is of niet. Conceptcursussen kunnen gebruikt worden om een cursus voor te bereiden voor die beschikbaar wordt voor studenten. Een conceptcursus kan ook gebruikt worden als sjabloon of als vragendatabank om inhoud te kopiëren naar andere cursussen. Een conceptcursus kan geen academiejaar hebben, en de instelling voor vragen en toegangsrechten kunnen ook niet veranderd worden.

Reviewer: Do we want to mention here that students cannot see draft courses?

> * `Conceptcursus`: dit bepaalt of de cursus een concept is of niet. Conceptcursussen kunnen gebruikt worden om een cursus voor te bereiden voor die beschikbaar wordt voor studenten. Het kan ook gebruikt worden om de cursus als sjabloon te gebruikt of als vragendatabank om inhoud te kopiëren vanuit andere cursussen. Een conceptcursus kan geen academiejaar hebben, and de instelling voor vragen en toegangsrechten kunnen ook niet veranderd worden.

Reviewer: ```suggestion
* `Conceptcursus`: dit bepaalt of de cursus een concept is of niet. Conceptcursussen kunnen gebruikt worden om een cursus voor te bereiden voor die beschikbaar wordt voor studenten. Een conceptcursus kan ook gebruikt worden als sjabloon of als vragendatabank om inhoud te kopiëren naar andere cursussen. Een conceptcursus kan geen academiejaar hebben, en de instelling voor vragen en toegangsrechten kunnen ook niet veranderd worden.
```
